conda_smithy/ci_register.py

```diff
@@ -217,6 +218,56 @@ def _encrypt_binstar_token(slug, item):
     return travis.encrypt(slug, item.encode()).decode('utf-8')
 
 
+def travis_configure(user, project):
+    """Configure travis so that it skips building if there is no .travis.yml present."""
+    headers = {
+               # If the user-agent isn't defined correctly, we will recieve a 403.
+               'User-Agent': 'Travis/1.0',
+               'Accept': 'application/vnd.travis-ci.2+json',
+               'Content-Type': 'application/json'
+               }
+    endpoint = 'https://api.travis-ci.org'
+    url = '{}/auth/github'.format(endpoint)
+    data = {"github_token": github.gh_token()}
+    response = requests.post(url, json=data, headers=headers)
+    if response.status_code != 201:
+        response.raise_for_status()
+
+    token = response.json()['access_token']
+    headers['Authorization'] = 'token {}'.format(token)
```

Comment:
Agreed. I think we should even aim to reuse the token over multiple calls. Possibly over multiple `conda-smithy` runs.